README.md

```diff
@@ -24,23 +24,44 @@
-
-- Set the ChaosEngine CR name as ENV (CHAOSENGINE) 
-  - For CR spec, see: https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosengine.yaml
-
-- If the experiments are not executed, apply the ChaosResult CRs manually 
-  - For CR spec, see: https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosresult.yaml
-
-- Run the exporter container (litmuschaos/chaos-exporter:ci) on host network. It is necessary to mount the kubeconfig
-  & override entrypoint w/ `./exporter -kubeconfig <path>`
-
-- Execute `curl 127.0.0.1:8080/metrics` to view metrics
+#### Pre-requisites:
+
+- A Working Local Kubernetes Cluster (Eg: Minikube or Vagrant)
+  - Set each of these Custom Resource Definition in your Kubernetes Cluster
+  - For ChaosEngine : https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosengine_crd.yaml
+  - For ChaosResult : https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosresults_crd.yaml
+  - For ChaosExperiment: https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosexperiment_crd.yaml
+  For information on these Custom Resources, please check this link : https://docs.litmuschaos.io/docs/next/co-components.html
+- Kube-config path of your local Kubernetes Cluster
+- `$GOPATH` set to your working directory
+
+### Further Steps: 
+
+The following steps are required to create sample chaos-related custom resources in order to visualize the metrics gathered by the chaos exporter
+
+- Clone this repo into your $GOPATH/litmuschaos"
+  `git clone https://github.com/litmuschaos/chaos-exporter`
+- Set an APP_UUID in the `~/.bashrc` or the `~/.profile`, add this command to set a default
+- Now, start your Local Cluster, (this guide helps in `minikube` but can be used for other offline clusters as well)
+- Create Kubernetes CR's(Custom Resources) for the litmus operator, link down below:
+- Now, as you have created the CustomResourceDefinition, Now it time to create the CustomResources for these definition above:
+  - For Default ChaosEngine : https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosengine.yaml
+    NOTE THAT THIS CHAOSENGINE COMES WITH A DEFAULT NAME ASSIGNED WITH IT WHICH IS : `engine-nginx`  you would need this afterwards
+  - For Default ChaosResult : https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosresult.yaml
+  - For the Default Experiments (Pod Delete Experiment) : https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosexperiment.yaml
+- As you have created the ChaosEngine, now again make changes in the `~/.bashrc` or the `~/.profile`, and the add this statement `export CHAOSENGINE=engine-nginx`, if you have changed the ChaosEngine name, then make those changes here as well.
+- Execute the command `source ~/.bashrc` or `source ~/.profile` according to the file you made changes in
```

Comment:
This seems too much steps around setup. Do we need this? Should we mention this?

Reply:
These steps are for mocking experiment runs in the cluster in order to see sample metrics. Should these be placed in a separate file which can be referred from README instead? while keeping the readme brief @AmitKumarDas

Reply:
If this is for starters and not contributors, I think this knowledge would be needed by them. Still these steps can be squashed by providing link to repo, and not the separate links.
Let me know, if this can improve the doc.